src/test/integration/configuration.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I would suggest renaming this file as configuration-example.json or something similar. Then adding configuration.json to .gitignore. This will help prevent devs in the future from committing this file and credentials by mistake.